README.rst

```diff
@@ -1,5 +1,27 @@
-ReadMe
-######
+openedx-translations
+####################
 
-The openedx-translations repository contains translations files from Open edX repos to be kept in sync with Transifex.
+The openedx-translations repository contains translations files from Open edX
+repositories to be kept in sync with Transifex. To accomplish this task, a GitHub Action
+in `.github/workflows/` named `extract-translation-source-files.yml` regularly extracts
+english translation source files form Open edX repositories containing code and adds them
+to this repository. A GitHub Transifex app allows for the automatic upload of these
+translation files and after being translated on Transifex, the automatic download back
+into this repository. The translation files in this repository can then be accessed by
+using the openedx-atlas CLI tool to download specific directories of translation files
```

Comment:
This should have a link to openedx-atlas.

Reply:
Added the link